vmb/matrimony/forms.py

```diff
@@ -176,6 +180,8 @@ class Meta:
             "name",
             "spiritual_name",
             "dob",
+            "actual_dob",
+            "gender",
```

Comment:
Users should not be able to change `gender` just like that. It should be done via a support request.

The `gender` field in `admin` can be made editable.